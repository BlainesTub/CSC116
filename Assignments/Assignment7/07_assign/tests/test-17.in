sin(x)^2 + cos(x)^2
1.5