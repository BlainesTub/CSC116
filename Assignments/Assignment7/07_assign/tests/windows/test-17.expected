Enter a function f(x): Enter a value of x: 
f(x) = sin(x)^2 + cos(x)^2
f(1.5) = 1
f'(x) = 2*cos(x)*1*sin(x)^1 + 2*-1*sin(x)*1*cos(x)^1
f'(1.5) = 0
